### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by pablen**

### CODE REVIEW

File: src/components/TermsAndConditions.js [hunk 1]
```diff
@@ -3,7 +3,10 @@ import React from 'react'
 
 const TermsAndConditions = ({ ParagraphComponent: P }) => (
   <React.Fragment>
-    <P>Copyright 2018 Autonomous Software</P>
+    <P>
+      Copyright 2018&ndash;
+      {new Date().getFullYear()} Autonomous Software
```
Comment (gabmontes) [inline src/components/TermsAndConditions.js]: Current year should not be dynamic but set at build time.

**Review by martincarrera — DISMISSED** [review]

File: src/selectors.js [hunk 2]
```diff
@@ -569,6 +556,32 @@ export const getChainsWithBalances = createSelector(
...
+  getConfig,
+  // eslint-disable-next-line max-params
+  (coinBalance, metBalance, chainsBalances, activeChain, isOnline, config) =>
+    config.enabledChains.length > 0
+      ? isOnline
+        ? utils.hasFunds(coinBalance)
+          ? utils.hasFunds(metBalance)
+            ? chainsBalances.filter(
+                chain =>
+                  chain.id !== activeChain && utils.hasFunds(chain.coinBalance)
```
Comment (gabmontes) [inline src/selectors.js:575]: Will this scale when the wallets gets support for a 3rd chain?

**Review by gabmontes — CHANGES_REQUESTED** [review]
The copyright change might be either fixed or moved to its own PR to merge the other changes.

File: src/selectors.js (hunk 2 above)
Comment (pablen) [reply src/selectors.js:575]: Yes, this will only disable the Port button if all possible destinations have no funds.

Comment (pablen) [reply src/selectors.js:575]: When we get support for a 3rd chain we will require more specific validations inside the Port form itself and check for funds after a destination chain is selected. That would need further thought and some UI changes so I think this check is enough for now.

**Comment by pablen** [comment]
Understood. Reverting this change.

**Review by gabmontes — APPROVED** [review]

**Review by martincarrera — APPROVED** [review]

**Pull request merged**